Fix: Schema validation for the correct definition of n..m relationships - must be non-nullable lists and entries (#736)
neo4j/graphql: docs/modules/ROOT/pages/type-definitions/relationships.adoc

    }
}
----

== Cardinality

> Neo4j GraphQL enforces that relationships defined as 'many' so given this example: 

Reviewer: ```suggestion
The Neo4j GraphQL Library has type definition requirements for "many" relationships, so given this example: 
```

[source, graphql, indent=0]
----
type User {
    name: String!
    posts: [Post!]! @relationship(type: "HAS_POST", direction: OUT)
}

type Post {
    name: String!
}
----

> the relationship at `User.posts` is considered a many relationship. Relationships such as the one above should always be of type NonNullListType and NonNullNamedType meaning both the array and the type inside of it should have a `!`.

Reviewer: ```suggestion
The relationship at `User.posts` is considered a "many" relationship. Relationships such as the one above should always be of type `NonNullListType` and `NonNullNamedType`, meaning both the array and the type inside of it should have a `!`.
```
